routes/api.php

```diff
@@ -13,7 +13,7 @@
     /** @var \Illuminate\Routing\Router $router */
 
     // ======== Contents ========
-    $router->get('contents', 'ContentController@index');
+    $router->get('contents', 'ContentController@index')->name('contents');
```

Comment:
I see why you needed that name here, but we're not using named routes in api right now, right? We'd need to guarantee that all those names are unique in whole app, so probably better would be to use something like `api.contents`.

BTW What was wrong with this approach `apiUrl("contents?updated_at=$yesterday,$tomorrow"`?

Reply:
* I have prefixed the api route names to with `api.`
* I wrote tests checking the output from the `route(...)`